ingestion-beam/src/test/java/com/mozilla/telemetry/integration/BigQueryIntegrationTest.java

```diff
@@ -368,7 +368,7 @@ public void canLoadRawFormat() throws Exception {
   }
 
   @Test
-  public void canLoadDecodedFormat() throws Exception {
+  public void canLoadAndReadDecodedFormat() throws Exception {
```

Comment:
I gave this existing test more responsibility rather than factoring out a separate test, since this already provides the necessary logic for putting realistic decoded bytes rows into BigQuery, which is exactly the format we need to exercise the new code.